e2e/cli-e2e/tests/collect.e2e.test.ts

```diff
@@ -3,7 +3,7 @@ import { executeProcess, readJsonFile, readTextFile } from '@code-pushup/utils';
 
 describe('CLI collect', () => {
   const exampleCategoryTitle = 'Code style';
-  const exampleAuditTitle = 'Require `const` declarations for variables';
+  const exampleAuditTitle = 'Require `const`';
```

Comment:
Why not keep the full title? :raised_eyebrow:

Reply:
Oh, I see. But that should no longer be the case once you revert the width back to original, right?

Reply:
As it was quite tricky to understand the error, is there a better way? RegEx or so, that is more stable to line breaks

Reply:
A regex allows you to use `\s` instead of a space. But that wouldn't handle the next column in between. A `.*` would cover it, but that's also quite likely to match something it shouldn't. 

I wouldn't complicate it. As long as we pick an audit title that fits on one line, then we're fine. This was is pretty short.

Reply:
When I was debugging it, `Disallow unused variables` worked for me.